src/atomate2/vasp/schemas/ferroelectric.py

```diff
@@ -0,0 +1,104 @@
+"""Schemas for Ferroelectric wflow."""
+
+from typing import List
+
+from pydantic import BaseModel, Field
+from pymatgen.core import Structure
+
+__all__ = ["PolarizationDocument"]
+
+
+class PolarizationDocument(BaseModel):
+    """Symmetry data set for materials documents."""
+
+    formula_pretty: str = Field(
+        None,
+        title="Pretty Formula",
+        description="Cleaned representation of the formula",
+    )
+
+    wfid: str = Field(None, title="WF id", description="The workflow id")
+
+    task_label_order: List[str] = Field(
+        None,
+        title="Task Labels Ordered",
+        description="Task labels in order from nonpolar to polar",
+    )
+
+    polarization_change: List[float] = Field(
+        None,
+        title="Polarization Vector",
+        description="The polarization vector",
+    )
+
+    polarization_norm: float = Field(
+        None,
+        title="Polarization Vector Norm",
+        description="The norm of the polarization vector",
+    )
+
+    same_branch_polarization: List[List[float]] = Field(
+        None,
+        title="Same Branch Polarization Vectors",
+        description="Polarization vectors in the same branch",
+    )
+
+    raw_electron_polarization: List[float] = Field(
+        None,
+        title="Raw Electron Polarization",
+        description="Electronic contribution to the polarization",
+    )
+
+    raw_ion_polarization: List[float] = Field(
+        None,
+        title="Raw Ions Polarization",
+        description="Ionic contribution to the polarization",
+    )
+
+    polarization_quanta: List[List[float]] = Field(
+        None,
+        title="Polarization Quanta",
+        description="Quanta of polarization for each structure and direction",
+    )
+
+    zval_dict: dict = Field(
+        None,
+        title="Atomic Z values",
+        description="Charge of the atoms as in pseudopotentials",
+    )
+
+    energies: List[float] = Field(
+        None,
+        title="Energies",
+        description="Total energy of each structure",
+    )
+
+    energies_per_atom: List[float] = Field(
+        None, title="Total energy per atom of each structure", description=""
+    )
+
+    outcars: List[dict] = Field(
```

Comment:
Avoid storing the full outcars, if there is specific information you want, then store that. Otherwise the uuid of the jobs will be sufficient to retrieve the full outcar information later.

Reply:
done